worlds/ladx/ItemIconGuessing.py

```diff
@@ -0,0 +1,197 @@
+FORBIDDEN = [
+    "TRADING",
+    "ITEM",
+    "BAD",
+    "SINGLE",
+    "UPGRADE",
+    "BLUE",
+    "RED",
+    "NOTHING",
+    "MESSAGE",
+    "POWER",
+    "MAGIC",
+    "PEGASUS",
+    "TAIL",
+    "ANGLER",
+    "FACE",
+    "BIRD",
+    "SLIME",
+    "GOLD",
+    "MAX",
+    "UPGRADE",
+    "BOMBS",
+    "ARROWS",
+    "NIGHTMARE",
+    "STONE",
+]
+
+SYNONYMS = {
+  'ANKLET': 'POWER_BRACELET',
+  'ARMLET': 'POWER_BRACELET',
+  'BAND': 'POWER_BRACELET',
+  'BANGLE': 'POWER_BRACELET',
+  'BRACER': 'POWER_BRACELET',
+  'CARRY': 'POWER_BRACELET',
+  'CIRCLET': 'POWER_BRACELET',
+  'CROISSANT': 'POWER_BRACELET',
+  'GAUNTLET': 'POWER_BRACELET',
+  'GLOVE': 'POWER_BRACELET',
+  'RING': 'POWER_BRACELET',
+  'STRENGTH': 'POWER_BRACELET',
+  'AEGIS': 'SHIELD',
+  'AEGISSHIELD': 'SHIELD', # Final Fantasy 1
+  'BUCKLER': 'SHIELD',
+  'SHLD': 'SHIELD',
+  'BALLISTA': 'BOW',
+  'GRAPPLE': 'HOOKSHOT',
+  'GRAPPLING': 'HOOKSHOT',
+  'ROPE': 'HOOKSHOT',
+  'BEAM': 'MAGIC_ROD',
+  'CANE': 'MAGIC_ROD',
+  'STAFF': 'MAGIC_ROD',
+  'WAND': 'MAGIC_ROD',
+  'BOOT': 'PEGASUS_BOOTS',
+  'GREAVES': 'PEGASUS_BOOTS',
+  'RUN': 'PEGASUS_BOOTS',
+  'SHOE': 'PEGASUS_BOOTS',
+  'SHOES': 'PEGASUS_BOOTS',
+  'SPEED': 'PEGASUS_BOOTS',
+  'FLUTE': 'OCARINA',
+  'RECORDER': 'OCARINA',
+  'JUMP': 'FEATHER',
+  'PLUME': 'FEATHER',
+  'WINGS': 'FEATHER',
```

Comment:
Ideally we should check for both "TESTWORD" and "TESTWORDS" in the list, to not have to manually mark up each plural word